tasmota/tasmota_xsns_sensor/xsns_54_ina226.ino

```diff
@@ -78,7 +78,7 @@
 
 #define INA226_REG_CONFIG                       (0x00)    // Config register
 #define INA226_RES_CONFIG                       (0x4127)  // Config register at reset
-#define INA226_DEF_CONFIG                       (0x42FF)  // Our default configuration
+#define INA226_DEF_CONFIG                       (0x47FF)  // Our default configuration
```

Comment:
Can you please take the opportunity of that change to enclose that #define line the #ifndef/#endif as suggested in the discussion
That will allow anyone to override that value if needed without touching the tasmota base code